hig.js

```diff
@@ -79,25 +81,40 @@ function runSpeech() {
 	const running = true;
 	var out = [];
 	genInterview(out);
-	while (speak(out)) {}
+	while (speak(out, os)) {}
 }
 
 function speak(phrases) {
 	if (phrases.length == 0) {
 		return false;  	
 	}
 	var p = phrases.shift();
-	var v = 'Alex';
-	if (p.name == ANNOUNCER) {
-		v = 'Samantha';
-	} 
+	if(os == 'mac'){
+		var v = 'Alex';
+			if (p.name == ANNOUNCER ) {
+				v = 'Samantha';
+		} 
+	}
+	else{
```

Comment:
What about linux or some other os, does linux use David?

Reply:
Also this is supposed to have a female voice for the announcer. is there no female voice on Windows that you can use?

Reply:
Yeah, I can definitely add more. The Windows voices are not working by passing in their names but will need to figure that out.  Ill work on resolving the issues you mention plus adding better OS handling.  Thanks!!

Reply:
According to this project's ReadMe https://github.com/xinggsf/say.js--TTS and other sources online, Windows just uses its default narrator.  Also, Linux has support but it looks like I need to make installations which I have not done because I have a Windows machine and cannot test it at the moment.  I will keep researching ways around these issues.

Reply:
This is technically still in progress but it does not seem like it is possible to set a voice besides the default voice with Windows

Reply:
I have an attempt at trying to time it up using the asynchronous model but it is not good enough yet. The synchronous way works at least and the delay is better than it was this morning.  Ive been trying to figure out how to time these phrases up by research, planning and minor experimentation because I have some ideas that require editing the json file and such and I do not want to do a ton of work if there is an easier way to accomplish it.